### PR TITLE
Some improvements for ida plugin

### PR DESCRIPTION
**Pull request by wtdcode**
1. Split the huge `Qiling.py` to several smaller files.
2. Fix the problem mentioned in #509.

Docs is on the way!

### CODE REVIEW

**Comment by wtdcode** [comment]
Docs: https://github.com/qilingframework/rtfd.io/pull/13

File: qiling/extensions/idaplugin/utils.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+from qiling.arch.x86_const import reg_map_misc as x86_reg_map_misc
+from qiling.arch.x86_const import reg_map_st as x86_reg_map_st
+from qiling.arch.arm_const import reg_map as arm_reg_map
+from qiling.arch.arm64_const import reg_map as arm64_reg_map
+from qiling.arch.mips_const import reg_map as mips_reg_map
+import collections
+from enum import Enum
+
+QilingHomePage = 'https://www.qiling.io'
+QilingGithubVersion = 'https://raw.githubusercontent.com/qilingframework/qiling/master/qiling/__version__.py'
```
Comment (xwings) [inline qiling/extensions/idaplugin/utils.py:20]: QilingVersion or just Version nothing to do with github

File: qiling/extensions/idaplugin/dialogs.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+class QlEmuAboutDialog(Form):
+    def __init__(self, version):
+        super(QlEmuAboutDialog, self).__init__(
+            r"""STARTITEM 0
+BUTTON YES* Open Qiling Website
+Qiling:: About
+            {FormChangeCb}
+            Qiling IDA plugin v%s, using Qiling Engine v%s.
+            (c) Qiling Team, 2020.
```
Comment (xwings) [inline qiling/extensions/idaplugin/dialogs.py:70]: (c) is bad for us :) take that out.

**Comment by xwings** [comment]
Thanks !

**Pull request merged**